src/SFA.DAS.EmploymentCheck.Data/Repositories/ApiOptionsRepository.cs

```diff
@@ -9,7 +9,6 @@ namespace SFA.DAS.EmploymentCheck.Data.Repositories
 {
     public class ApiOptionsRepository : IApiOptionsRepository
```

Comment:
This class is being flagged by SonarCube as having no code coverage - as it calls the Azure Storage it's probably not testable so can be marked with the ExcludeFromCodeCoverage attribute